Sources/YUL/Functions.swift

```diff
@@ -0,0 +1,219 @@
+//
```

Comment:
Don't comment out an entire file, delete it. We have source control should we ever need to refer to its contents again.